src/codemodder/codemods/base_visitor.py

```diff
@@ -23,6 +23,7 @@ def __init__(
     def filter_by_result(self, node):
         pos_to_match = self.node_position(node)
         if self.results is None:
+            # Some codemods must run without results existing.
             return True
```

Comment:
I documented this because it was strange to return True here (do we filter by result or not filter by result?). Then I realized the logic here is that this method is used for all codemods - those with and those without results. We still want to analyze codemods even if there are no results. Maybe later on we can have a `filter_by_result` that's more relevant. The one I implemented for jwt sonar returns False here, since there should always be results if we want to run the codemod.

Reply:
Yes I encountered this too. We're using `None` as a sentinel for codemods that do not have detectors, and we do not want to short-circuit those cases. If you don't mind, could you update the comment to say something along those lines?